things/postgres/init.go

```diff
@@ -83,16 +83,14 @@ func migrateDB(db *sqlx.DB) error {
 				Id: "things_2",
 				Up: []string{
 					`ALTER TABLE IF EXISTS things ALTER COLUMN
-					 metadata TYPE JSONB using metadata::text::jsonb
-					`,
+					 metadata TYPE JSONB using metadata::text::jsonb`,
```

Comment:
I kinda liked that this comma was in the new line. I am not saying that this is good solution, but I think it would be worth to check best practices when writing SQL strings in the code...

Reply:
I would follow the standard which is to don't add a new line. We are also following this standard in other services.

Reply:
@manuio I agree, I just suggested to check where this standard commens and if it widely accepted, not something just invented by us. How other big Go projects handle this problem?

Reply:
I saw 2 versions:
```
				Up: []string{
					`ALTER TABLE IF EXISTS channels ALTER COLUMN
					 metadata TYPE JSONB using metadata::text::jsonb`,
				},
```
or 
```
				Up: []string{`
                                         ALTER TABLE IF EXISTS channels ALTER COLUMN
					 metadata TYPE JSONB using metadata::text::jsonb`,
				},
```

Reply:
I was expecting some research links... And my remark was about comma (`,`) at the end of the statement, not backticks.

Reply:
@drasko  the `,` still there

Reply:
Is it, but it is not in the new line. So this is what I am trying to figure out - should it be in a new line or not.

Reply:
In this case it's not a good proposition since it doesn't build

Reply:
How did it build before then?

Reply:
Because of the ``,`